sagemathinc/cocalc
# Typescript projects

# Description
Typescript new project creation and filter buttons for hidden and deleted projects.

# Testing Steps
1. Create a new project
1. Delete the project
1. Check the deleted button
1. Hide the project
1. Check the hidden button and the deleted button

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):
